README.md

```diff
@@ -8,13 +8,13 @@
 yarn
 yarn start
 ```
-Then open `http://localhost:8080`. Hot reloading of both react and static files (ejs, sass) files should work. Sometimes gulp forgets to watch new files so you need to rerun `yarn start`.
+Then open `http://localhost:9090`. Hot reloading of both react and static files (ejs, sass) files should work. Sometimes gulp forgets to watch new files so you need to rerun `yarn start`.
 
 ### Environment variables
 
 In `.env.example` your will find all used env variables. When you do `yarn start` this file will be copied as `.env` (only if it doesn't exists already). If you want to change something modify `.env` file which is gitignored. If you want to add new env variable make sure to add default to `.env.example`.
 
-FAQ_ENABLED - 1 to display faq
+FAQ_ENABLED - 1 to display faq  
```

Comment:
Those spaces will move env variables to new lines. IMO that's more readable.

Reply:
Ah yeah they render incorectly. Can you try using list instead:
```
- FAQ_ENABLED - 1 to display faq
- PLATFORM_ENABLED - 1 to display platform subpage```

Reply:
Ofc but what is wrong with using two spaces? It's part of [markdown spec](http://markdown-guide.readthedocs.io/en/latest/basics.html#line-return)

Reply:
tbh i wasnt aware of it oO I dont like it that much because it formats much differently then its raw form. IMHO list is better because...well it's list of env vars.